userprofile/api/serializers.py

```diff
@@ -1,8 +1,7 @@
 from rest_framework import serializers
-
```

Comment:
new line here would be nice to keep external libraries and project imports separated.